src/Sarif.Multitool/Rules/SarifValidationSkimmerBase.cs

```diff
@@ -6,6 +6,7 @@
 using System.Linq;
 using System.Resources;
 using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.Writers;
```

Comment:
>Writers [](start = 35, length = 7)

Is this a remnant of a change you decided not to make? Since there are no other changes, and it built before, this can't be necessary (unless, I suppose, you changed the namespace of some other class that this class uses).